parisienne/block-template-parts/contact.html

```diff
@@ -0,0 +1,19 @@
+<!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:image {"id":351,"sizeSlug":"large"} -->
+<figure class="wp-block-image size-large"><img src="https://user-images.githubusercontent.com/5375500/71100587-4af19480-2183-11ea-9749-7df2d4e4c9ad.png" alt="" class="wp-image-351"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:jetpack/contact-form {"subject":"Let's collaborate!","to":"jeff.ong@automattic.com","hasFormSettingsSet":"yes"} -->
```

Comment:
For the initial demo, can we replace with with something else? That'll remove the reliance on Jetpack, and eliminate a setup step for everyone. For now, we could just populate this page with the fake address, phone number, and email or something.

Reply:
I debated about this also, and partially wanted to feel the friction that is having to install Jetpack and make sure the contact block module is enabled.

I agree we should remove its dependence for ease of example though.